Backpack/src/main/res/values-night/internal.semantic.color.xml

```diff
@@ -21,4 +21,10 @@
   <color name="__textFieldHint">@color/bpkTextTertiaryDark</color>
   <color name="__textFieldIcon">@color/bpkTextSecondary</color>
   <color name="__bottomNavUnselected">@color/bpkTextSecondaryDark</color>
+
+  <color name="__barChartBarBackgroundColor">@color/bpkSkyGrayTint01</color>
```

Comment:
Is using sky gray the correct for dark mode?

Reply:
Yes, it supposed to have low contrast ratio with the background.